src/VectorizedSolver.cpp

```diff
@@ -322,76 +328,52 @@ void VectorizedSolver::normalizeFluxes() {
 void VectorizedSolver::computeFSRSources() {
 
   int tid;
-  FP_PRECISION scatter_source;
-  FP_PRECISION fission_source;
-  FP_PRECISION* nu_sigma_f;
-  FP_PRECISION* sigma_s;
-  FP_PRECISION* sigma_t;
-  FP_PRECISION* chi;
   Material* material;
+  FP_PRECISION* sigma_t;
+  FP_PRECISION* sigma_s;
+  FP_PRECISION* fiss_mat;
+  FP_PRECISION scatter_source, fission_source;
 
-  int size = _num_FSRs * _num_groups * sizeof(FP_PRECISION);
+  int size = _num_threads * _num_groups * sizeof(FP_PRECISION);
   FP_PRECISION* fission_sources = (FP_PRECISION*)MM_MALLOC(size, VEC_ALIGNMENT);
-  size = _num_threads * _num_groups * sizeof(FP_PRECISION);
   FP_PRECISION* scatter_sources = (FP_PRECISION*)MM_MALLOC(size, VEC_ALIGNMENT);
 
   /* For all FSRs, find the source */
-  #pragma omp parallel for private(material, nu_sigma_f, chi, \
-    sigma_s, sigma_t, fission_source, scatter_source) schedule(guided)
+  #pragma omp parallel for private(tid, material, sigma_t, sigma_s, fiss_mat, \
+    fission_source, scatter_source) schedule(guided)
```

Comment:
Thanks for the help @jtramm! I just pushed an update which used your approach for both ``CPUSolver::computeFSRSources()`` and ``VectorizedSolver::computeFSRSources()``. Although this doesn't address @geogunow's concern about memory limitations, it does address the false sharing issue which I alluded to above.

I tried to apply this approach to allocate the ``thread_fsr_fluxes`` array in the ``CPUSolver::transportSweep()`` once for each thread rather than over and over again for each track, but was unable to get it working. Sometimes the OpenMP is like voodoo to me. That is outside the scope of this PR, but might be something to look at it in the future (albeit it should hardly affect performance at all).